### ISSUE TITLE
Test classes naming is not detected properly

### ISSUE THREAD

**Issue by sbcgua** [abaplint/abaplint#501]
code
```abap
class ltcl_mockup_loader_mock definition final
  for testing.

  public section.
  ...
endclass.
```
abaplint.json contains the rule
```
    "local_class_naming": {
      "enabled": true,
      "exclude": [],
      "local": "^LC(L|X)_.*$",
      "test": "^LTCL_.*$"
    },
```
resulting error
```
src/...clas.testclasses.abap[1, 7]  - Local class naming, expected ^LC(L|X)_.*$ (local_class_naming)
```

**Comment by larshp** [comment]
https://github.com/sbcgua/mockup_loader/commit/3d5bec0fdba87fce223c0f6e19c68f143a242514#annotation_56694250

**Comment by larshp** [comment]
try again, 1.8.0 published

**Comment by sbcgua** [comment]
works well ! Thanks ! :+1:
